README.md

```diff
@@ -1,4 +1,5 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/esta/issues)
```

Comment:
You should change (https://github.com/dwyl/esta/issues) to (https://github.com/DevOpsVisions/EasyApp/issues)

Reply:
Hi @MohamedRadwan-DevOps you have a good eye ! thanks for your replay i changing the url to the backlog of our community.

Reply:
Thank you @Wajiho, much appreciate your time and effort!